### ISSUE TITLE
UnboundLocalError: local variable 'picotts_command' referenced before assignment

### ISSUE THREAD

**Issue by Santobert** [rhasspy/rhasspy#167]
Running version 2.5.9 with docker-compose. My main device (no sound IO) won't launch after upgrading to version 2.5.9.
Do you need more specific information? I am happy to test or provide more logs.

```python
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
   "__main__", mod_spec)
 File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/lib/rhasspy/rhasspy-supervisor/rhasspysupervisor/__main__.py", line 109, in <module>
    main()
  File "/usr/lib/rhasspy/rhasspy-supervisor/rhasspysupervisor/__main__.py", line 89, in main
    mosquitto_path=args.mosquitto_path,
  File "/usr/lib/rhasspy/rhasspy-supervisor/rhasspysupervisor/__init__.py", line 183, in profile_to_conf
    mqtt_password=mqtt_password,
  File "/usr/lib/rhasspy/rhasspy-supervisor/rhasspysupervisor/__init__.py", line 2545, in print_text_to_speech
    mqtt_password,
  File "/usr/lib/rhasspy/rhasspy-supervisor/rhasspysupervisor/__init__.py", line 2094, in get_text_to_speech
    shlex.quote(" ".join(str(v) for v in picotts_command)),
UnboundLocalError: local variable 'picotts_command' referenced before assignment
```

**Comment by ayavilevich** [comment]
I also have this error, but when trying to save settings

```
[ERROR:2021-01-18 08:56:33,281] rhasspyserver_hermes: local variable 'picotts_command' referenced before assignment
Traceback (most recent call last):
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1821, in full_dispatch_request
    result = await self.dispatch_request(request_context)
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1869, in dispatch_request
    return await handler(**request_.view_args)
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__main__.py", line 991, in api_profile
    return save_profile(profile_json)
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__main__.py", line 404, in save_profile
    new_profile, supervisor_conf_file, local_mqtt_port=_ARGS.local_mqtt_port
  File "/usr/lib/rhasspy/rhasspy-supervisor/rhasspysupervisor/__init__.py", line 183, in profile_to_conf
    mqtt_password=mqtt_password,
  File "/usr/lib/rhasspy/rhasspy-supervisor/rhasspysupervisor/__init__.py", line 2545, in print_text_to_speech
    mqtt_password,
  File "/usr/lib/rhasspy/rhasspy-supervisor/rhasspysupervisor/__init__.py", line 2094, in get_text_to_speech
    shlex.quote(" ".join(str(v) for v in picotts_command)),
UnboundLocalError: local variable 'picotts_command' referenced before assignment
```

How can I try an older version?

**Comment by synesthesiam** [comment]
Should be fixed in the latest 2.5.9 update :+1:
